src/rules/noVarRequiresRule.ts

```diff
@@ -52,11 +66,19 @@ class NoVarRequiresWalker extends Lint.ScopeAwareRuleWalker<{}> {
     public visitCallExpression(node: ts.CallExpression) {
         const expression = node.expression;
 
+        const options = this.getOptions() as any[];
+        const patternConfig = options[0] as { "ignore-module": string } | undefined;
+        const ignorePattern = patternConfig === undefined ? undefined : new RegExp(patternConfig[OPTION_IGNORE_MODULE]);
+
         if (this.getCurrentDepth() <= 1 && expression.kind === ts.SyntaxKind.Identifier) {
             const identifierName = (expression as ts.Identifier).text;
             if (identifierName === "require") {
```

Comment:
you can add an additional condition to ensure there is only one argument. that way you don't have to loop through the arguments below